examples/server_sync.py

```diff
@@ -283,8 +284,9 @@ def get_commandline():
         args.slaves = 0
     if not args.framer:
         args.framer = comm_defaults[args.comm][0]
-    if not args.port:
-        args.port = comm_defaults[args.comm][1]
+    args.port = args.port or comm_defaults[args.comm][1]
+    if args.comm in set("tcp", "udp", "tls"):
```

Comment:
Same as before please use comm_defaults.

Reply:
This is a slight typo, I will have it fixed. I am missing the inner `list` inside. The problem with `port` expected to  be int is that I can not pass a port of choice when trying to spin a serial server . If that is what is expected then there is no point of having a command line option to set port.

Reply:
there are a reason,  server_sync is common for multiple transports and the command line is shared.

Reply:
Exactly, then it should be intuitive enough to be used across serial and tcp based protocols with out much fuss.